### PR TITLE
utils_disk: new added parts

### PR DESCRIPTION
**Pull request by dzhengfy**
This is to provide a new utitily function which is very basic and
can be invoked by many other utitily functions.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Review by chunfuwen — APPROVED** [review]
lgtm

**Comment by dzhengfy** [comment]
@Yingshun could you help review? Thanks.

File: virttest/utils_disk.py [hunk 1]
```diff
@@ -1102,6 +1102,20 @@ def get_parts_list(session=None):
     return parts
 
 
+def get_added_parts(session, old_parts):
```
Comment (Yingshun) [inline virttest/utils_disk.py:1105]: How about setting a default value for session? I think this function can also be used in localhost, just like get_parts_list(). What do you think?

Comment (dzhengfy) [reply virttest/utils_disk.py:1105]: My original thought is that we will not test scenario to add a new disk to a host. So I only apply this function to operate in vm.

Comment (Yingshun) [reply virttest/utils_disk.py:1105]: Ok, I see. But I still think it's better to use the same way as get_parts_list(). Anyway, it's up to you!

Comment (dzhengfy) [reply virttest/utils_disk.py:1105]: I would not like to write codes which we do not expect to use. However, I will be happy to enhance it once we need it in future. Thanks.

Comment (Yingshun) [reply virttest/utils_disk.py:1105]: Ok, got it.

**Review by Yingshun — APPROVED** [review]

**Pull request merged**